apps/nova/src/components/TutorProfile/DeleteRating.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useCallback, useMemo } from "react";
+import {
+  Alert,
+  AlertType,
+  Dialog,
+  toaster,
+  useFormatMessage,
+} from "@litespace/luna";
+import { useDeleteRatingTutor } from "@litespace/headless/rating";
+import { Void } from "@litespace/types";
+import { useInvalidateQuery } from "@litespace/headless/query";
+import { QueryKeys } from "@litespace/headless/constants";
+
+type DeleteRatingProps = {
+  open: boolean;
+  close: Void;
+  id: number;
+  tutorId: number;
+};
+
+const DeleteRating: React.FC<DeleteRatingProps> = ({
+  id,
+  tutorId,
+  open,
+  close,
+}) => {
+  const invalidate = useInvalidateQuery();
+  const intl = useFormatMessage();
+
+  const onSuccess = useCallback(() => {
+    toaster.success({ title: intl("tutor.rate.delete.success") });
+    invalidate([QueryKeys.FindTutorRating, tutorId]);
+    close();
+  }, []);
+  const onError = useCallback((error: Error) => {
+    toaster.error({
+      title: intl("tutor.rate.delete.error"),
+      description: error.message,
+    });
+    close();
+  }, []);
+
+  const deleteRating = useDeleteRatingTutor({ onSuccess, onError });
+
+  const action = useMemo(() => {
+    return {
+      label: intl("global.labels.confirm"),
+      onClick: () => deleteRating.mutate(id),
+      disabled: deleteRating.isPending,
+      loading: deleteRating.isPending,
+    };
+  }, [intl, deleteRating]);
```

Comment:
Please add empty line

```suggestion
  }, [intl, deleteRating]);
  
```